schedpolicy/sched_policy.go

```diff
@@ -19,30 +19,3 @@ type SchedulePolicy interface {
 	// SchedPolicyGet returns schedule policy matching given name.
 	SchedPolicyGet(name string) (*SchedPolicy, error)
 }
-
-func NewDefaultSchedulePolicy() SchedulePolicy {
```

Comment:
If we removed default ```Not Implemented```  implementation, caller will be working with in-memory fake implementation of schedule policy. Is that expected behaviour?

Reply:
before, the caller was getting a panic, because there was no implementation. The one you see above which was removed was the one in the REST server, but no in the ClusterManager.